tests/monkeypatch/test_llama_attn_hijack_flash.py

```
@@ -50,6 +50,19 @@ def test_get_max_seqlen_in_batch(self):
         target_res = torch.tensor([4, 3, 5, 2], dtype=torch.int32)
         self.assertTrue(torch.allclose(get_max_seqlen_in_batch(attn_mask), target_res))
 
+    def test_get_max_seqlen_in_batch_adjacent_rows_same_id(self):
+        # Rows whose first/last segment ids match across the row boundary must not
+        # be merged into a single segment (each row is a fresh pack).
+        attn_mask = torch.tensor([[1, 1, 1, 1], [1, 1, 1, 1]])
+        target_res = torch.tensor([4, 4], dtype=torch.int32)
+        self.assertTrue(torch.allclose(get_max_seqlen_in_batch(attn_mask), target_res))
+
+        indices, cu_seqlen, max_seqlen_in_batch = get_unpad_data(attn_mask)
```

Ruff (0.15.20)

[warning] Unpacked variable `indices` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)